.vscode/launch.json

```diff
@@ -12,6 +12,8 @@
 			"args": [
 				"--disable-extension",
 				"asvetliakov.vscode-neovim",
+				"--disable-extension",
+				"pokey.cursorless-local",
```

Comment:
This is my current hack to avoid auto-update if you're running a fork.  Just change the extension id on your local fork to `cursorless-local`.  For some reason the version number thing didn't work 🤷‍♂️